ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-OCM-274.yaml

```diff
@@ -0,0 +1,35 @@
+binary_build_commands: make install
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- dockerfile_path: build/ci-tf-e2e.Dockerfile
+  to: ocm-tf-e2e
+releases:
+  latest:
+    integration:
+      name: "4.13"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-presubmit
```

Comment:
refinement of pre-submit are gone..

Reply:
I'll keep it in place before merging it to the ``main`` config